Update to RTN-005 for actual working practices (#27)
lsst/rtn-005: body.tex

\section{Introduction}

This document provides a guide to the \VRO\ approach to work management and annual planning.
See the operations proposal \citeds{RDO-018} for a description of the full scope and high level goals of the program.

> There is no formal \gls{EVMS} required from the funding Agencies (the \gls{NSF} and the \gls{DOE} Office of Science) so activities are planned in detail on a semi-annual basis and effort towards that schedule of activities is tracked using agile tools.

Reviewer: ```suggestion
There is no formal \gls{EVMS} required from the funding Agencies (the \gls{NSF} and the \gls{DOE} Office of Science) so activities are planned in detail on an annual basis and effort towards that schedule of activities is tracked using agile tools.
```

Reviewer: I don't think we are planning activities semi-annually; more like annual  setting of milestones in May-August and then epic planning in Jira in September (and then supplemented through the year as needed).

The Head of \gls{Operations} is Ranpal Gill and the Program Coordinator is Cathy Petry.
They monitor the budget and maintain details within the \gls{NOIRLab} accounting system.
They assist in developing the annual \gls{POP}, tracking milestones and reporting on progress.

On the \gls{SLAC} side, Christine Soldahl is the \gls{Business Manager} who handles similar tasks.

> \gls{Rubin Operations} has two operational Departments and one Program in addition to the Director’s Office: \gls{RDO}), \gls{RSO}, \gls{RDM}, and \gls{REO}; a Program within Rubin). Each operational Department is led by an \gls{AD}; REO is led by a Head of Program).

Reviewer: ```suggestion
\gls{Rubin Operations} has two operational Departments and one Program in addition to the Director’s Office: \gls{RDO}), \gls{RSO}, \gls{RDM}, and \gls{REO}, a program within Rubin). Each operational Department is led by an \gls{AD}; REO is led by a Head of Program).
```

\subsection{Responsibility and Authority}

> The \glspl{AD} and Head of \gls{REO} are given broad authority to organize, manage, and staff their areas within an overall budget and operations plan \citedsp{RDO-018} managed by the \gls{RDO}. Annual budgets and plans are developed through a scrub process led by the Program Operations team of the \gls{RDO}. This process informs \gls{NSF} and \gls{DOE} side budgets and plans for the upcoming fiscal year. Budgets and plans are captured in the NOIRLab \gls{POP} and the \gls{SLAC} \gls{FWP} submitted to \gls{NSF} and \gls{DOE}, respectively.

Reviewer: ```suggestion
The \glspl{AD}s and Head of \gls{REO} are given broad authority to organize, manage, and staff their areas within an overall budget and operations plan \citedsp{RDO-018} managed by the \gls{RDO}. Annual budgets and plans are developed through a scrub process led by the Program Operations team of the \gls{RDO}. This process informs \gls{NSF} and \gls{DOE} side spending and work plans for the upcoming fiscal year. Budgets and plans are captured in the NOIRLab \gls{POP} and the \gls{SLAC} \gls{FWP} submitted to \gls{NSF} and \gls{DOE}, respectively.
```

Reviewer: We don't scrub the budget for the upcoming year, we scrub the spending plan within that budget

> The \glspl{AD} and Head of \gls{REO} are delegated the responsibility for detailed staffing plan, activities, and major milestones to be delivered in their respective areas as well as other aspects of the plan as assigned by the \gls{RDO}. Staffing roles that are vacant is the direct responsibility of the \glspl{AD} and Head of \gls{REO} (or their delegates). Changes to existing roles or promotions/new roles must be approved by the Director or their delegate (usually the Deputy Director). Recruitments are to be initiated by the \glspl{AD} and Head of \gls{REO} (or their delegate) and coordinated with the Program Operations team. Program Operations will work with NOIRLab to complete the recruitment once approved by the Director.

Reviewer: ```suggestion
The \glspl{AD} and Head of \gls{REO} are delegated the responsibility for detailed staffing plan, activities, and major milestones to be delivered in their respective areas as well as other aspects of the plan as assigned by the \gls{RDO}. Staffing roles that are vacant are the direct responsibility of the \glspl{AD} and Head of \gls{REO} (or their delegates). Changes to existing roles or promotions/new roles must be approved by the Director or their delegate (usually the Deputy Director). Recruitments are to be initiated by the \glspl{AD} and Head of \gls{REO} (or their delegate) and coordinated with the Program Operations team. Program Operations will work with NOIRLab or SLAC to complete the recruitment once approved by the Director or Deputy Director.
```

  & 1.1 & \hspace{0.5cm} \gls{Director}'s Office  \tabularnewline
  & 1.2 & \hspace{0.5cm} \gls{Safety}  \tabularnewline
  & 1.3 & \hspace{0.5cm} Program \gls{Operations}\footnotemark[1]  \tabularnewline
  & 1.4 & \hspace{0.5cm} In-Kind Program Coordination  \tabularnewline
  & 1.5 & \hspace{0.5cm} Legacy Survey of Space and Time  \tabularnewline
  & 1.6 & \hspace{0.5cm} Change and Risk Management  \tabularnewline
  & 1.7 & \hspace{0.5cm} Special Investigations  \tabularnewline
2 & & Rubin Summit \gls{Operations}  \tabularnewline
  & 2.1 & \hspace{0.5cm} \gls{RSO} Management  \tabularnewline
  & 2.2 & \hspace{0.5cm} Summit Science  \tabularnewline
  & 2.3 & \hspace{0.5cm} Summit Software  \tabularnewline
  & 2.4 & \hspace{0.5cm} Summit Facility  \tabularnewline
  & 2.5 & \hspace{0.5cm} Nighttime \gls{Operations}  \tabularnewline
  & 2.6 & \hspace{0.5cm} Summit Engineering  \tabularnewline
  & 2.7 & \hspace{0.5cm} Camera  \tabularnewline
  & 2.8 & \hspace{0.5cm} Summit Systems Engineering  \tabularnewline
3 & & Rubin Data Management \tabularnewline
  & 3.1 & \hspace{0.5cm} \gls{RDM} Management  \tabularnewline
  & 3.2 & \hspace{0.5cm} Data Facilities  \tabularnewline

>   & 3.5 & \hspace{0.5cm} Summit DevOps  \tabularnewline

Reviewer: ```suggestion
  & 3.5 & \hspace{0.5cm} Chilean DevOps  \tabularnewline
```

  & 3.6 & \hspace{0.5cm} Pipeline Middleware  \tabularnewline
  & 3.7 & \hspace{0.5cm} Data Engineering  \tabularnewline
  & 3.8 & \hspace{0.5cm} Algorithms and Pipelines  \tabularnewline
  & 3.9 & \hspace{0.5cm} Campaign Management  \tabularnewline
  & 3.10 & \hspace{0.5cm} Service Quality and Reliability Engineering  \tabularnewline
  & 3.11 & \hspace{0.5cm} QServ  \tabularnewline
  & 3.12 & \hspace{0.5cm} Build Engineering  \tabularnewline
  & 3.13 & \hspace{0.5cm} Community Science  \tabularnewline
  & 3.14 & \hspace{0.5cm} Data Validation  \tabularnewline
5 & & \gls{REO} \tabularnewline
  & 5.1 & \hspace{0.5cm} Education and Public Outreach  \tabularnewline

\hline

\end{longtable}

%\subsection{The Control Account Manager}
%\label{sec:cam}

Detailed work is planned in advance of each fiscal year at the team level.

> Team leads will work with department \glspl{AD} to develop plans for activities that address specific milestones, projects, and level of effort activity.

Reviewer: ```suggestion
Team leads will work with their department's \glspl{AD} to develop plans for activities that address specific milestones, projects, and level of effort activity.
```

% Note: These calculations are true for Construction but need to be updated for Rubin Operations since NOIRLab uses 1744 hours per year (83%) for scientists, scientists could have 20% or 50% for research, and research time and Training, Admin & DEI time is charged to RSS.  For Engineers, Training, Admin & DEI time is charged to ES.
% We could also refer to the effort planning tool Phil has made ("Team Effort Planner" in Staffing Plan).  Right now, most people have "Commenter" permission in the STaffing Plan workbook and this is not sufficient to use the dropdown filter that TLs and ADs would need to use this tool.  We could move this to a stand-alone workbook.

> \gls{Rubin Operations} uses 1,726 hours per year as a fiducial value for effort estimation purposes. This figure is \emph{after} all vacations, sick leave, etc are taken into account. \gls{SLAC} may have different definitions for tracking their staff time. This value is used to estimate effort for activities and deliverables at the department level through planing epics and determines FTE needs.

Reviewer: ```suggestion
\gls{Rubin Operations} at NOIRLab uses 1,726 hours per year as a fiducial value for effort estimation purposes. This figure is \emph{after} all vacations, sick leave, etc are taken into account. \gls{SLAC} may have different definitions for tracking their staff time. This value is used to estimate effort for activities and deliverables at the department level through planing epics and determines FTE needs.
```

In general, staff in \gls{Rubin Operations} roles at a given expected full-time equivalent (\gls{FTE}) effort level are expected to devote that fraction of their total work time to \RO.

% Remove Feb 2025
% Staff in ``scientist'' or ``engineer'' roles can allocate up to 2\% of their time to training, 2\% to administrative activities and 1\% to outreach and \gls{DEI} activities.

Staff in ``scientist'' roles spend either 20\% (259 hours) or 50\% (777 hours) of their time on personal research (see the \citeds{RDO-018} Plan for details).

% Confirmation on how we will be having staff charge for ES/RSS budgeted time in FY22 will be needed.  One suggestion is to not worry about this until FY23 when staff are contributing more than 50% of their time to NOIRLab.
% Removed DEI time

> Personal research time is charged to a \gls{NOIRLab}'s \gls{RSS} project code and is prorated for staff who are fractionally allocated to \gls{Rubin Operations}.

Reviewer: ```suggestion
At NOIRLab, personal research time is charged to a \gls{NOIRLab}'s \gls{RSS} project code and is prorated for staff who are fractionally allocated to \gls{Rubin Operations}.
```

Training, administrative, and outreach time is charged to either \gls{RSS}, \gls{ES}, \gls{DMS}, \gls{CEE} depending on where staff are allocated into \gls{NOIRLab}.

> Functional Managers together with Program Operations will ensure the proper project codes are available on timecards.

Reviewer: ```suggestion
At all institutions, functional supervisors together with Program Operations will ensure the proper project charge codes are available on timecards.
```

Note that specific activities related to ``onboarding'' and getting up to speed with operations can be ticketed as regular work.

> For example, working through tutorials, reading documentation, and so on are all activities which can earn \glspl{SP}.

Reviewer: ```suggestion
This includes, for example, working through tutorials, reading documentation, and so on.
```

\fi

\iffalse
\subsubsection{Team Leads and other Leadership Roles}

> Individuals in leadership roles may find it necessary to assign a larger fraction of their time to \gls{LOE} type work, and therefore spend fewer hours generating \glspl{SP}.

Reviewer: ```suggestion
Individuals in leadership roles may find it necessary to assign a larger fraction of their time to \gls{LOE} type work.
```

> The ratio of hours to \glspl{SP} remains constant, but the number of hours decreases.

Reviewer: ```suggestion
% The ratio of hours to \glspl{SP} remains constant, but the number of hours decreases.
```

   \begin{itemize}

>       \item Semi-annual process of planning epics and L3s for next six months

Reviewer: ```suggestion
      \item Semi-annual process of planning epics and L3s for the next twelve months
```

   \end{itemize}
\item April-May: \gls{NOIRLab} \gls{POP} process and \gls{SLAC} \gls{FWP} development kick off mid-April, with major milestones and budget due end of May.
   \begin{itemize}
      \item Early April: set next \gls{FY} major milestones
         \begin{itemize}
            \item \glspl{AD} review and/or add/edit major milestones
            \item Discuss in Ops Exec
            \item Implement changes in \gls{JIRA}
         \end{itemize}
      \item Late April: set next \gls{FY} (and \gls{FY}+2) budget:
         \begin{itemize}
            \item Early April: freeze planning tools (\gls{WBS} and Cost Calculator)
            \item \gls{JOR} - timing can vary
            \item \gls{RDO} issues departmental budget guidance based on costed and reviewed plan.
         \end{itemize}

>       \item May: next \gls{FY} Sandboxing:

Reviewer: ```suggestion
      \item May: next \gls{FY} Scrubbing:
```

         \begin{itemize}

>             \item Early May: Prepare and issue sandbox workbooks, one per department.

Reviewer: ```suggestion
            \item Early May: re-issue the Scrub Sandbox workbook.
```

            \item Early May Department management teams facilitate team-level sandbox pitches (including L2/L3 milestones, changes to spending plans)

>             \item Late May: Sandboxing Workshop, for all ops teams and \gls{Director}'s Office.

Reviewer: ```suggestion
            \item Late May: Scrub Workshop, for all ops teams and \gls{Director}'s Office.
```

         \end{itemize}
   \end{itemize}

> \item June-July: implement sandboxed changes.

Reviewer: ```suggestion
\item June-July: implement sandboxed proposals.
```

   \begin{itemize}

>       \item Implement proposed changes to Staffing Plan, \gls{Risk} Register, \gls{JIRA} L2/L3 milestones

Reviewer: ```suggestion
      \item Directorate approves proposed changes, iterating with the ADs.
      \item Program Ops implements proposed changes to Staffing Plan, \gls{Risk} Register, \gls{JIRA} L2/L3 milestones
```

      \item End of June: Final Labor and Nonlabor budget for \gls{NOIRLab} \gls{POP}
      \item Update lab-wide budget models (\gls{RAS}) from Staffing and Nonlabor plans, for quarterly forecasting.
      \item July: derive next \gls{FY} contract and \gls{MPO} \glspl{SOW}.
   \end{itemize}
\item September
   \begin{itemize}
      \item Update lab-wide budget models (\gls{RAS}) from Staffing and Nonlabor plans, for quarterly forecasting.

>       \item Semi-annual process of planning epics and L3s for next six months

Reviewer: ```suggestion
      \item Annual work planning in epics for the next twelve months
```

\end{description}

Some of these milestones are exposed to external reviewers; it is important that these be delivered on time and to specification.
Level 1 and 2 milestones are under change control once they are defined and described in a \gls{JIRA} Milestone issue. Any change there after must follow the defined Change Control process.

> Level 3 milestones are defined for use within Departments and not required to go under project change control, but properly adhering to the plan is important: other teams will use these milestones to align their schedules, so they rely on accuracy.

Reviewer: ```suggestion
Level 3 milestones are defined by Team Leaders for use within Departments, and not required to go under observatory-wide change control, but properly adhering to the plan is important: other teams will use these milestones to align their schedules, so they rely on accuracy.
```

Epics should work to achieve milestones, i.e., they may be blocking issues on the milestones.
When a detailed description of work for a given \gls{epic} is known, it is described in \gls{JIRA}.
It should then be assigned to the appropriate \glspl{cycle}.

> Progress is tracked against milestones in issues linked to \gls{JIRA} epics. For activities with defined start and end dates, progress is initially estimated based on elapsed time—for example, an activity scheduled over six months would be assumed to be 50% complete after three months. This time-based progress is then verified by the Team Lead and may be adjusted to reflect the actual status of the activity.

Reviewer: ```suggestion
Progress is tracked against milestones in issues linked to \gls{JIRA} epics. For activities with defined start and end dates, progress is initially estimated based on elapsed time—for example, an activity scheduled over six months would be assumed to be 50\% complete after three months. This time-based progress is then verified by the Team Lead and may be adjusted to reflect the actual status of the activity.
```

> To ensure success, \gls{JIRA} epics must be completely detailed out prior to a full 6-month \gls{cycle} and start and due dates must be estimated.

Reviewer: ```suggestion
To ensure success, \gls{JIRA} epics must be completely detailed out prior to a full 12-month \gls{cycle} and start and due dates must be estimated.
```

\subsection{Defining the Schedule with Milestones}

> Rubin Milestones are defined as \gls{JIRA} issues of type ``milestone''. As indicated above, the \gls{Director} (or their designate) defines the L1 milestones, the \glspl{AD} define their departments' L2 milestones, and the Team Leaders and \glspl{AD} define the L3 milestones for their teams.

Reviewer: ```suggestion
Rubin Milestones are defined as \gls{JIRA} issues of type ``milestone.'' As indicated above, the \gls{Director} (or their designate) defines the L1 milestones, the \glspl{AD}s define their departments' L2 milestones, and the Team Leaders and \glspl{AD}s define the L3 milestones for their teams.
```

Each \gls{epic} loaded into the plan must have this minimum set of fields filled in:

\begin{itemize}
\item A concrete, well defined deliverable \emph{or} be clearly described as a ``bucket'' or ``emergent work'' (\secref{sec:bucket});
\item The \texttt{Component} field set to the appropriate Department;
\item In the \texttt{Linked Issues} field, create a \textit{``blocks''} link to the parent milestone to reflect how that milestone contributes to achieving that milestone.
\item The \texttt{Start Date} field set to the appropriate date, which is not earlier than the start date of the milestone it is labeled to achieve.
\item The \texttt{Due Date} field set to the appropriate date, which does not exceed the due date of the milestone it is labeled to achieve.
\item The \texttt{label} field is set to identify the fiscal year during which the work will be done. Examples are \gls{FY23} or \gls{FY24}.

\end{itemize}
The fields above are required to have values entered because they define the tracking of progress. Other fields in the \gls{epic} can also be filled in as needed.

Be aware that:

\begin{itemize}

> \item An \gls{epic} may only be assigned to a single \gls{cycle}.

Reviewer: ```suggestion
\item An \gls{epic} may only be assigned to a single L3 milestone.
```

>   It is not possible to define an \gls{epic} that crosses the \gls{cycle} boundary (see \secref{sec:cycle-close} for the procedure when an \gls{epic} is not complete by the end of the \gls{cycle}).

Reviewer: ```suggestion
  It is possible to define an \gls{epic} that crosses the \gls{cycle} boundary (see \secref{sec:cycle-close} for the procedure when an \gls{epic} is not complete by the end of the \gls{cycle}).
```

Reviewer: I think we can and do have epics that lead to milestones in future years, no?

> It is the responsibility of technical managers to ensure that new tickets are handled appropriately, updating the schedule to include them where necessary.

Reviewer: ```suggestion
It is the responsibility of Team Leaders or their designated technical leads to ensure that new tickets are handled appropriately, updating the schedule to include them where necessary.
```

It is required that the \texttt{Team} field be set to the appropriate team (\jira{RFC-145}).
%Both RFCs mentioned above a very old from 2016 - i'm not sure they are still current with out set up?

> This indicates which manager is responsible for seeing that the work is completed successfully.

Reviewer: ```suggestion
This indicates which Team Leader is responsible for seeing that the work is completed successfully.
```

Reviewer: Is it clear throughout this document that the relevant managers are the Team Leaders, who are the functional supervisors for all their team members? We use the term "manager" a lot in this document, but in ops we usually talk about Team Leaders instead.

Available teams, and the associated managers, are listed in the \href{https://developer.lsst.io/processes/jira_agile.html}{Developer Guide}; generally speaking, they align with the the work breakdown structure described in \secref{sec:wbs}.

> Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Management” team may be used to indicate that the \gls{AD} of Data Management is responsible for assigning the work.

Reviewer: ```suggestion
Where there is uncertainty about which team should be responsible for a particular ticket, the department management team may be used to indicate that the relevant \gls{AD} is responsible for assigning the work.
```

\subsection{Tracking Progress toward Milestones}
\label{sec:track-progress}

%Progress on completing epics is visualized in Smartsheet.
%Smartsheet lists all Level 1 through Level 3 milestones in a gantt-chart style view.
Each Level 1 milestone is achieved by completing a series of Level 2 and/or Level 3 milestones.
Progress is tracked against milestones in issues linked to JIRA epics. For activities with defined

> start and end dates, progress is estimated based on elapsed time

Reviewer: ```suggestion
start and end dates, progress is estimated based on elapsed time.
```

It also describes our approach to risk-aware budgeting, and a process for reviewing and approving requests beyond baseline targets.

\subsection{The annual budget planning cycle}

Figure~\ref{fig:annual-cycle} illustrates the phases of \gls{Rubin Operations} budget development and spending planning throughout the year, as laid out in Section~\ref{sec:annual-planning-timeline}.
In April, following several months collecting high level and external budget information and implementing changes requested during the previous summer, the \gls{Director}'s Office sets major (Level 1 and some Level 2) milestones, and issues departmental budget guidance for the next two fiscal years.

> (Ideally, the \gls{JOR} would be held in early April, to provide external expert oversight of the plan.)

Reviewer: ```suggestion
(The \gls{JOR} provides external expert oversight of the plan, and is held either in April or September to provide feedback on either the pre-scrub budget or the post-scrub spending plan, respectively.)
```

The major milestones and budget are inserted into the draft \gls{NOIRLab} \gls{POP} and \gls{SLAC} \gls{FWP}.
(Only the budget for the coming fiscal year is required for the \gls{POP}.)

In May, the Departments and Teams hold a series of meetings to set their minor (remaining Level 2 and Level 3) milestones, and ``scrub'' their budgets, milestones, and risks, in order to propose their next-year spending plans (to the \gls{Director}'s Office) by the end of May.

> We provide each department with a ``sandbox'' workbook to work up during this period.

Reviewer: ```suggestion
We provide the departments and teams with a ``scrub sandbox'' workbook to work up during this period.
```

> We initialize this sandbox with the department's labor and non-labor plans (that were used in setting the budget), the current staffing plan, the progress against milestones in the current fiscal year, and the major milestones for the coming fiscal year.

Reviewer: ```suggestion
We initialize this sandbox with all the teams' labor and non-labor plans (that were used in setting the budget), the current staffing plan, the progress against milestones in the current fiscal year, and the major milestones for the coming fiscal year.
```

> We then invite them to ``scrub'' their plans, by working this sandbox up with analysis of recent performance, and any changes they want to propose for the next fiscal year, based on the minor milestones and associated activities they want to prioritize.

Reviewer: ```suggestion
We then invite the Team Leaders to ``scrub'' their plans, by working this sandbox up with analysis of recent performance, and any changes they want to propose for the next fiscal year, based on the minor milestones and associated activities they want to prioritize.
```

We also invite them to scrub their part of the risk register, update their risk response plans in concert with their planned activities, and anticipate any \glspl{RBT} they may need to make.

> Typically, each team works on its part of the sandbox mostly independently, and then presents their scrubbed plan to the department's management team around mid-May in a ``pitch'' session.

Reviewer: ```suggestion
Typically, each team works on its part of the sandbox mostly independently, and then presents their scrubbed plan to the department's management team in a mid-month check-in.
```

> (We provide a template slide deck for each team's pitch, to ensure that all aspects of the sandboxing/scrubbing exercise are addressed.)

Reviewer: ```suggestion
(We provide a template slide deck for the teams to use at the scrub workshop and mid-month check-in, to ensure that all aspects of the sandboxing/scrubbing exercise are addressed.)
```

The teams then address the feedback they have been given by the management team, and then they present their scrubbed plans to the other teams in the department, the Directorate and Program \gls{Operations} teams from the \gls{Director}'s Office, and the teams in the other departments, at the sandbox workshop at the end of the month.
This sandbox workshop at the end of May / beginning of June gives all the departments, including the \gls{Director}'s Office, a chance to review, and probe, the ground-up planning in each team.
(In practice, only the three technical departments' management teams are likely to sit through the entire workshop, although it is organized inclusively so that anyone could choose to review all the scrubbed plans.)

In most cases a team's proposed changes in effort distribution, staffing, and purchasing will either involve only small deviations from budget guidance, or stay within the ``box'' given to them.
Teams may also coordinate their proposed changes so as to keep the department's overall spending plan within its budget.
(This coordination may be facilitated by the department's management team, e.g.\ following the pitch sessions.)

> Following the Sandbox Workshop, the \gls{Director}'s Office adjusts and finalizes the overall spending plan by the end of June, in time for the submission of the NOIRLab \gls{POP}, and \gls{SLAC}'s \gls{DOE} budget briefing.

Reviewer: ```suggestion
Following the Sandbox Workshop, the \gls{Director}'s Office approves, adjusts and finalizes the overall spending plan by the end of June, in time for the submission of the NOIRLab \gls{POP}, and \gls{SLAC}'s \gls{DOE} budget briefing.
```

This June period of post-sandbox spending plan development and cost calculation involves the Directors Office staff making updates to the planning tools (Appendix~\ref{sec:planning-tools}), with review by the departmental management teams.

We implement the changes to the spending plan, once captured in the Rubin planning tools, in \gls{NOIRLab}'s and \gls{SLAC}'s lab-wide budget models in July.
Statements of work for the next fiscal year's contracts and \glspl{MPO} are straightforwardly derived from the scrubbed plans, for their July deadlines.

> In September, the teams then plan work (for the first half of the new fiscal year) against their minor milestones, incorporating any community input they may have collected at the annual Community Workshop in August.

Reviewer: ```suggestion
In September, the teams then plan work (for the first half of the new fiscal year) against their minor milestones, incorporating any community input they may have collected at the annual Community Workshop at the end of July.
```

The ground-up spending plan developed during the annual \gls{cycle} is to cover operations work defined by the milestone activities, as well as any regular work captured in bucket epics.
This work includes mitigation of risks held by the teams, as described in the response plans in the risk register.
Any risks that are realized during the year will require action that is not budgeted for.
Additional spending to address realized risks is enabled by Rubin's \gls{RBT} process, whereby reserve funds are released for use by the departments in response to a specific mid-year request made in a custom Jira ticket.

Standard terminology for financial reserves is that ``management reserve'' describes funds held back to pay for the unknown unknowns, while ``contingency reserve'' is used for known unknowns (i.e.\ the risks in the risk register).
Rubin \gls{Operations}' two funding agencies offer differing guidelines on how reserve should be managed.

> \gls{DOE} allows reserves to be built up and carried forward at \gls{SLAC}, and used as both contingency and management reserve.

Reviewer: ```suggestion
\gls{DOE} allows a limited reserve to be built up and carried forward at \gls{SLAC}, and used as contingency reserve for the specific risk of delayed or reduced funding (e.g. resulting from a government shutdown).
```

> \gls{DOE} \gls{HEP} holds additional reserves centrally, which can also be used as both contingency and management reserve.

Reviewer: ```suggestion
\gls{DOE} \gls{HENP} holds additional reserves centrally, which can also be used as both contingency and management reserve.
```

Author: HENP is not an existing acronym - what does it stand for?

Author: assuming it is High Energy and Nuclear Physics

Currently, the total cost exposure, and hence the needed level of contingency reserve, is expected to be approximately 15\% of the total annual operating budget at each operations partner.
Included in the risk register is the risk of federal funding allocations being reduced, with a likely cost (if realized in any given year) of some \$1.5M per operations partner.

> (At \gls{SLAC}, for example, this would cover about 7\% of the operating expense -- enough to pay the labor bills during a 1 month funding allocation delay.) The top risks (high probability and at least moderate severity) should be reported regularly to the funding agencies so that they can plan for the possibility of providing supplemental funds to address these risks if they are realized.

Reviewer: ```suggestion
(At \gls{SLAC}, for example, this would cover about 7\% of the operating expense -- enough to pay the labor bills during a 1 month funding allocation delay.) The top risks (high probability and at least moderate severity) need to be reported regularly to the funding agencies so that they can plan for the possibility of providing supplemental funds to address these risks if they are realized.
```
